docroot/themes/humsci/humsci_basic/src/js/shared/equal-height-grid/index.js

```diff
@@ -13,6 +12,7 @@ const applyStretchClass = () => {
 
   Array.prototype.forEach.call(cardCollections, (collection) => {
     const verticalLinkedCards = [...collection.querySelectorAll('.hb-vertical-linked-card')];
+    const verticalLinkedCardTitles = [...collection.querySelectorAll('.hb-vertical-linked-card__title')];
```

Comment:
I still don't see a purpose of the spread operator. `querySelectorAll` returns the array. why copy it?